test/e2e/adminapi_resources.go

```diff
@@ -59,17 +59,16 @@ var _ = Describe("[Admin API] List Azure resources action", func() {
 			subnets[strings.ToLower(*p.SubnetID)] = struct{}{}
 		}
 
-		vnet, err := clients.VirtualNetworks.Get(ctx, r.ResourceGroup, r.ResourceName, "")
+		vnet, err := clients.VirtualNetworks.Get(ctx, r.ResourceGroup, r.ResourceName, nil)
 		Expect(err).NotTo(HaveOccurred())
```

Comment:
Response from Get is not being processed.
```suggestion
		resp, err := clients.VirtualNetworks.Get(ctx, r.ResourceGroup, r.ResourceName, nil)
		Expect(err).NotTo(HaveOccurred())
		vnet := resp.VirtualNetwork
```

Reply:
ditto